docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -79,6 +79,14 @@ import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
 
 {{"demo": "LocalizationLuxon.js"}}
 
+:::warning
+Luxon adapter does not support `Settings.throwOnInvalid = true` [setting](https://moment.github.io/luxon/api-docs/index.html#settingsthrowoninvalid).
```

Comment:
Nitpick:

```suggestion
The Luxon adapter does not support `Settings.throwOnInvalid = true` [setting](https://moment.github.io/luxon/api-docs/index.html#settingsthrowoninvalid).
```

or

```suggestion
`AdapterLuxon` does not support `Settings.throwOnInvalid = true` [setting](https://moment.github.io/luxon/api-docs/index.html#settingsthrowoninvalid).
```

From what I see we rend to use `AdapterLuxon` most of the time

Reply:
Thanks for the suggestion, we definitely do. 👍